crates/ev-cli/Cargo.toml

```diff
@@ -16,4 +16,11 @@ human-panic = "1.0.3"
 log = "0.4.17"
 env_logger = "0.9.0"
 exitcode = "1.1.2"
+dialoguer = "0.10.2"
+attestation-doc-validation = "0.7.4"
```

Comment:
Would be nice to remove some deps like this, and move all EnclaveApi stuff into `ev-enclave`, but it's outside scope of this PR. Objective here is just to do what's necessary to move the cli commands themselves into `ev-cli`